docontrol/README.md

```diff
@@ -0,0 +1,75 @@
+# DoControl
+
+## Overview
+
+This integration allows [DoControl](https://www.docontrol.io/) customers to forward their DoControl related logs and events to Datadog through automated security workflows.
+
+## Setup
+
+To set up this integration, you must have an active [DoControl account](https://www.docontrol.io/demo). You must also have proper admin permissions in Datadog.
+
+### Installation
+
+No installation is required on your host.
+
+### Use Datadog actions in DoControl's workflows
+
+You must create a Datadog API key and an application key to use as input parameters for Datadog actions in DoControl.
+
+#### Create an API key in Datadog
+
+1. In Datadog, hover over your username in the bottom left corner and select **Organization Settings**.
+
+2. From the left panel, click **API Keys**.
+
+3. Click **+ New Key**.
+
+   ![Get_DD_API_Key](https://raw.githubusercontent.com/DataDog/integrations-extras/master/docontrol/images/Get_DD_API_Key.png)
```

Comment:
I don't think this screenshot is correct, as the yellow box is highlighting "log out" rather than "organization settings"